src/state/routing/slice.ts

```diff
@@ -114,12 +122,11 @@ export const routingApi = createApi({
           }
         )
       },
-      async queryFn(args, _api, _extraOptions, fetch) {
-        const { tokenInAddress, tokenInChainId, tokenOutAddress, tokenOutChainId, amount, routerPreference, type } =
-          args
-
-        try {
-          if (routerPreference === RouterPreference.API) {
+      async queryFn(args: GetQuoteArgs, _api, _extraOptions, fetch) {
```

Comment:
```suggestion
      async queryFn(args: GetQuoteArgs | SkipToken, _api, _extraOptions, fetch) {
```

Do you need to handle the skip token? I'm not sure if this is baked in, but widgets does so. See https://github.com/Uniswap/widgets/blob/main/src/state/routing/slice.ts#L58.